src/admin-views/settings/featured/links.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Help links for featured settings box.
+ *
+ * @since TBD
+ *
+ * @var Tribe__Tickets__Admin__Views $this   Template object.
```

Comment:
Here it's not an instance of `Tribe__Tickets__Admin__Views` it's an instance of `Tribe__Template`